src/components/notifications/Slack.vue

```diff
@@ -20,6 +20,9 @@
             <i18n-t tag="p" keypath="aboutWebhooks" style="margin-top: 8px;">
                 <a href="https://api.slack.com/messaging/webhooks" target="_blank">https://api.slack.com/messaging/webhooks</a>
             </i18n-t>
+            <p style="margin-top: 8px;">
+                {{ $t("aboutUsername", [$t("slack"), $t("Username"), $t("Friendly Name")]) }}
```

Comment:
I know that in other translations (in this file) this is done, but I don't like this pulling things out unless that makes translation simpler.

=> let's inline this one

```suggestion
                {{ $t("aboutSlackUsername") }}
```

Reply:
Indeed, it seems easier to manage.

(Also, when viewed in Japanese, it feels somewhat difficult to read since only parts are translated.)